packages/app-check/android/build.gradle

```diff
@@ -60,19 +60,32 @@ project.ext {
 }
 
 android {
+  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
+  if (agpVersion >= 7) {
+    namespace 'io.invertase.firebase.appcheck'
+  }
+
   defaultConfig {
     multiDexEnabled true
     // Here we add a build config field to allow devs to provide a
     // FIREBASE_APP_CHECK_DEBUG_TOKEN to be used in debug mode.
     buildConfigField "String", "FIREBASE_APP_CHECK_DEBUG_TOKEN", "\"${System.env.FIREBASE_APP_CHECK_DEBUG_TOKEN}\""
   }
+
+  buildFeatures {
+    // AGP 8 no longer builds config by default
```

Comment:
I encountered this on AnkiDroid but as I was contemplating implementing this shortly (you beat me to it, thank you) I would have forgotten about this. Great catch